README.md

```diff
@@ -87,23 +87,23 @@ This directory is bind mounted to both the **django** and **nginx** containers.
 
 #### Database
 Get the full stack up & running from inside the *summit_blood_samples* dir with
-`$ docker-compose -f production.yml up --build`
+`$ sudo docker-compose -f production.yml up --build`
```

Comment:
is the change to running as root needed because of systemd?

Reply:
No, it's the way the prod server is set up. Normal users don't have permissions to talk to the docker daemon, so sudo/root is needed to do anything.